packages/editor/src/components/plain-text/index.native.js

```diff
@@ -21,6 +21,12 @@ export default class PlainText extends Component {
 		}
 	}
 
+	componentDidUpdate( prevProps ) {
```

Comment:
Do we need this part? I've tested without this change and it seemed to work. Can you provide an example of a case this is needed to run, so I make sure I tested correctly? Thanks in advance

Reply:
Yes, just start the demo app on gb-mobile, and select the `Code` block, than tap on the `Page Break` block. 
If you comment this the keyboard is still up.
Same if you tap on the side of the `More` block, without editing the actual "more" text.